dask/array/tests/test_reductions.py

```diff
@@ -67,6 +67,8 @@ def test_reductions_1D(dtype):
     reduction_1d_test(da.nansum, a, np.nansum, x)
     with ignoring(AttributeError):
         reduction_1d_test(da.nanprod, a, np.nanprod, x)
+        reduction_1d_test(da.nancumprod, a, np.nancumprod, x)
```

Comment:
By this you mean just making sure I'm running with the master versions of numpy, xarray, and dask, right?  Once I did this the `xarray.prod` went away.